addons/html_editor/static/src/main/toolbar/toolbar.js

```diff
@@ -12,15 +12,29 @@ export class Toolbar extends Component {
                 // TODO: more specific prop validation for buttons after its format has been defined.
                 buttonGroups: Array,
                 buttonsActiveState: Object,
+                namespace: Object,
             },
         },
     };
 
     setup() {
         this.buttonsActiveState = useState(this.props.toolbar.buttonsActiveState);
+        this.namespace = useState(this.props.toolbar.namespace);
     }
 
     dispatch(cmd, payload) {
         this.props.toolbar.dispatch(cmd, payload);
     }
+
+    getFilteredButtonGroups() {
+        if (this.namespace.tag) {
```

Comment:
why namespace need tag ? namespace in command => string, namespce from props => { tag } look confusing

Reply:
in `toolbar_plugin.js` the namespace is defined as an object with the tag key:
```js
this.namespace = reactive({ tag: undefined });
```
in the `Toolbar` component it is passed as an object with the tag key.

The object with `tag` is not convenient but the problem is that we need a Set or a Map we can mutate to not lose the proxy. If you've any suggestion I'm open